packages/app/src/files-manager/components/files-manager/files-manager.interface.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { Type } from "Renderer/components/core/icon/icon.config"
+import { FilesType } from "App/files-manager/constants/files-manager.enum"
+
+export interface DiskSpaceCategory {
+  filesType: FilesType
```

Comment:
```suggestion
  fileType: FileType
```

Reply:
Changed in [5d0cd24](https://github.com/mudita/mudita-center/pull/598/commits/5d0cd24e83476e4283bfc8a768ebb6b9a844767c)